src/example-types/results.js

```diff
@@ -33,7 +33,7 @@ let convertPopulate = getSchema =>
 
       let $lookup = {
         $lookup: {
-          as,
+          as: x.as || as,
```

Comment:
I don't see the point of this change -- the existing code already lets you do this by making up a `fieldName`, putting it in the `include`s, and then using it as the key for a `populate` object, which then specifies the real field as `localField` -- there are examples of this all over the existing projects. It doesn't seem good to me to introduce a second way of doing the same thing

Reply:
@doug-patterson, Github opened the #92 which I closed 2 days ago. and caused confusion. 
as Sam discuss https://github.com/smartprocure/contexture-mongo/pull/92#issuecomment-758243898
although we've never actually liked that, we just never came up with something better